enterprise/enrollment_api.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to get details from the course catalog API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+from edx_rest_api_client.client import EdxRestApiClient
+
+
+def get_enrollment_api_client():
+    """
+    Retrieve a client of the Enrollment API.
+
+    The client is authenticated using the API token from the Django settings.
+    """
+    # TODO(smarnach): Find a useful way to configure the Enrollment API URL.
+    enrollment_api_url = 'http://{}/api/enrollment/v1'.format(settings.SITE_NAME)
+    return EdxRestApiClient(enrollment_api_url, oauth_access_token=settings.EDX_API_KEY, append_slash=False)
```

Comment:
The authentication does not work like this.  We need to pass a custom header instead.